src/Listener/SaveSuspensionToDatabase.php

```diff
@@ -61,7 +61,9 @@ public function whenUserWillBeSaved(UserWillBeSaved $event)
 
             $this->assertCan($actor, 'suspend', $user);
 
-            $user->suspend_until = new DateTime($attributes['suspendUntil']);
+            $user->suspend_until = $attributes['suspendUntil']
+                ? new DateTime($attributes['suspendUntil'])
```

Comment:
Also here, relates to my other remark in this review. We're using Carbon, please import that class and instantiate that instead of the DateTime object.